final_task/rss_reader/functions/check_func.py

```diff
@@ -0,0 +1,85 @@
+"""Contain checking functions
+
+Functions
+---------
+check_internet_connection(logger) -> Bool
+    Check if network connection is avaliable
+--------------------------------------------
+check_version_argument(command_line_args) -> None
+    If version argument, print version and exit
+--------------------------------------------
+check_verbose(command_line_args): -> logger
+    If vebose argument is True, set logger to info level
+--------------------------------------------
+check_feed_status(feed, logger)
+    Check HTTP status code
+--------------------------------------------
+def check_news_collection(news_collection, logger) -> news_collection
+    Check if news_collection is not empty
+"""
+
+import requests
+import logging
+import sys
+
+
+def check_internet_connection(logger):
+    """Check if network connection is avaliable"""
+    try:
+        logger.info("Checking internet connection...")
+        requests.get('http://216.58.192.142', timeout=1)
+        return True
+    except requests.ConnectionError:
```

Comment:
It is better to make application raise an exception and gracefully shutdown if there is no internet connection.

Reply:
Fixed